_config/ispn.yml

```diff
@@ -32,16 +32,16 @@ stable:
 # Please remove or comment out the following 'unstable' version section if this is not applicable at the moment.
 unstable:
    minor_version: 11.0
-   version: 11.0.0.Dev05
-   maven_latest: 11.0.0.Dev05
-   release_date: 2020/05/01
-   fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12345476
+   version: 11.0.0.CR1
+   maven_latest: 11.0.0.CR1
+   release_date: 2020/05/24
+   fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12344573
    codename: N/A
-   whats_new: Indexing improvements, non-blocking everywhere, etc...
+   whats_new: XSite IRAC, non-blocking everywhere, server security usability, etc...
```

Comment:
"XSite IRAC" => XSite/Cross Site improvements
I thought we are going to use the word "IRAC" only internally and in the code.